docs/source/howto/run_codes.rst

```diff
@@ -346,6 +347,28 @@ At the end, you receive a confirmation, with the *PK* and the *UUID* of your new
 
     Note: remove the ``--`` prefix and replace dashes (``-``) within the keys with an underscore ( ``_`` ).
 
+
+.. note::
+
+    It is possible to run codes that are provided by a `Conda environment <https://docs.conda.io/en/latest/>`_.
+    The code configuration YAML would look something like the following:
+
+    .. code-block:: yaml
+
+        filepath_executable: 'executable-name'  # This can simply be the executable name, no need for the absolute path
+        prepend_text: conda activate environment-name
+
+    Note that the configuration is not complete but only shows the relevant lines.
+    In the `prepend_text`, the required Conda environment is activated as one normally would.
+    This allows to simply specify the target executable by its name, and there is no need to specify an absolute path.
```

Comment:
You may consider skipping these two lines (they seem obvious from the example above)
```suggestion
```